codesign/lib/src/zip_archive.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:codesign/codesign.dart';
+import 'package:file/file.dart';
+
+/// A zip file that contains files that must be codesigned.
+abstract class ZipArchive extends ArchiveFile {
```

Comment:
Is this logic needed yet, or should we move it to a future PR?

Reply:
Yeah they are for later PRs. I removed them and the corresponding function declarations, and replaced with a TODO.